builders/toolkit/ethereum-api/precompiles/cross-chain-transfers.md

```diff
@@ -0,0 +1,228 @@
+2. Paste the file contents into a Remix file named `XcmInterface.sol`
+
+### Compile the Contract {: #compile-the-contract }
+
+Next, you will need to compile the interface in Remix:
+
+1. Click on the **Compile** tab, second from top
+2. Compile the interface by clicking on **Compile XcmInterface.sol**
+
+![Compiling XcmInterface.sol](/images/builders/toolkit/ethereum-api/precompiles/xcm-interface/xcm-interface-1.webp)
+
+When the compilation is completed, you will see a green checkmark next to the **Compile** tab.
+
+### Access the Contract {: #access-the-contract }
+
+Instead of deploying the precompile, you will access the interface given the address of the precompiled contract:
+
+1. Click on the **Deploy and Run** tab directly below the **Compile** tab in Remix. Please note that the precompiled contracts are already accessible at their respective addresses. Therefore, there is no deployment step
+2. Make sure **Injected Provider - Metamask** is selected in the **ENVIRONMENT** dropdown. Once you select **Injected Provider - Metamask**, you may be prompted by MetaMask to connect your account to Remix if it's not already connected
+3. Make sure the correct account is displayed under **ACCOUNT**
+4. Ensure **XCM - XcmInterface.sol** is selected in the **CONTRACT** dropdown. Given that it is a precompiled contract, there is no deployment step. Instead, you are going to provide the address of the precompile in the **At Address** field
+5. Provide the address of the precompile: `{{networks.dancebox.precompiles.xcmInterface}}` and click **At Address**
+
+![Access the address](/images/builders/toolkit/ethereum-api/precompiles/xcm-interface/xcm-interface-2.webp)
+
+The **XCM Interface** precompile will appear in the list of **Deployed Contracts**.
+
+### Send Tokens Over to Another EVM-Compatible Appchain {: #transfer-to-evm-chains }
+
+To send tokens over to an account in another EVM-compatible appchain, please follow these steps:
+
+1. Expand the **transferAssetsToPara20** function
+2. Enter the appchain ID (paraId)
+3. Enter the 20-bytes (Ethereum-like) destination account (beneficiary)
+4. Specify the tokens to be transferred. Note that this parameter is an array that contains at least one asset. Each asset is specified by its address and the total amount to transfer
+
+--8<-- 'text/builders/toolkit/ethereum-api/precompiles/xcm-interface/erc-20-note.md'
+
+5. Enter the index of the asset that will be used to pay the fees. This index is zero-based, so the first element is `0`, the second is `1`, and so on 
+6. Enter the maximum gas to pay for the transaction. This gas is derived from two parameters, the processing time (`refTime`) and the proof size (`proofSize`). In practice, setting refTime to `uint64::MAX` is equal to *unlimited weight*
+7. Click **transact**
+8. MetaMask will pop up, and you will be prompted to review the transaction details. Click **Confirm** to send the transaction
```

Comment:
Numbering starts over here for me, this can be avoided by indenting the note in line with the indentation amount of the numbered section. 

<img width="926" alt="Screenshot 2024-08-07 at 10 02 02 AM" src="https://github.com/user-attachments/assets/a3fa080d-baf5-4abf-a58f-eea8d3ba8194">